features/src/main/feature/camel-features.xml

```diff
@@ -765,6 +765,22 @@
         <bundle dependency='true'>mvn:commons-io/commons-io/2.15.1</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-exec/${project.version}</bundle>
     </feature>
+    <feature name='camel-elytron' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-http</feature>
+        <feature>jakarta-servlet</feature>
+        <bundle>mvn:org.apache.camel.karaf/camel-undertow/${project.version}</bundle>
```

Comment:
do we need `dependency='true'` for the bundles the `camel-elytron` depend on?

Reply:
yes, that's right thanks